docs/config.md

```diff
@@ -63,17 +63,6 @@ Secret('**********')
 '98n349$%8b8-7yjn0n8y93T$23r'
 ```
 
-Similarly, the `URL` class will hide any password component
```

Comment:
Should we instead teach how to create a `URL` class?

Reply:
We don't have the URL class anymore. I think in the sqlalchemy docs we can have better examples, I tried to keep this minimum.

Reply:
On our last conversation, I think we agreed to show the user how to do this, right?

Reply:
Yeah, based on our conversation I checked the databases code: https://github.com/encode/databases/blob/ab5eb718a78a27afe18775754e9c0fa2ad9cd211/databases/core.py#L420
The class has about 120 loc which I think is an overkill to include in the Starlette docs. I don't think Starlette's docs is the right place for this.

Reply:
what we talked is that we could just include the link of the code 👀

Reply:
Yeah I know, we wanted to see if it's small include it here, if it's big then link to it.
But giving it more thought I think it doesn't make sense to do in Starlette docs, another point is that the databases is hardly maintained that's why we are removing it.